ocfweb/stats/mirrors.py

```diff
@@ -37,14 +37,18 @@ def stats_mirrors(request: HttpRequest) -> HttpResponse:
 
 
 @periodic(86400)
-def bandwidth_semester() -> Tuple[Any, Any]:
+def bandwidth_semester(humanize=True) -> Tuple[Any, Any]:
```

Comment:
```suggestion
def bandwidth_semester(humanize: bool = True) -> Tuple[Any, Any]:
```

This parameter needs a type annotation.